scripts/cargo-install-all.sh

```diff
@@ -144,7 +144,10 @@ mkdir -p "$installDir/bin"
   if [[ -z "$validatorOnly" ]]; then
     # the patch-related configs are needed for rust 1.69+ on Windows; see Cargo.toml
     # shellcheck disable=SC2086 # Don't want to double quote $rust_version
-    "$cargo" $maybeRustVersion install --locked spl-token-cli --root "$installDir"
+    "$cargo" $maybeRustVersion \
+      --config 'patch.crates-io.ntapi.git="https://github.com/solana-labs/ntapi"' \
+      --config 'patch.crates-io.ntapi.rev="97ede981a1777883ff86d142b75024b023f04fad"' \
+      install --locked spl-token-cli --root "$installDir"
```

Comment:
oh, i see now. this lines are actually needed until spl-token-cli starts to use published solana monorepo crates which used the bumped crate.

for now, it's not the case...

this means i need further follow up pr to really remove patching in Cargo.locks..

Reply:
Ohhh, makes sense. Thanks for hunting that!